cmd/snap-update-ns/utils.go

```diff
@@ -482,7 +482,10 @@ func planWritableMimic(dir, neededBy string) ([]*Change, error) {
 			// The undo logic handles rbind mounts and adds x-snapd.unbind
 			// flag to them, which in turns translates to MNT_DETACH on
 			// umount2(2) system call.
-			Name: dir, Dir: safeKeepingDir, Options: []string{"rbind"}},
+			//
+			// The rprivate is there to ensure that changes made to the
+			// original directory do not propagate into this view.
+			Name: dir, Dir: safeKeepingDir, Options: []string{"rprivate", "rbind"}},
```

Comment:
Yes, you are correct. I simply expressed that the part we care about is inward propagation (because we don't mount anything in this view the outward propagation does not matter). I will adjust the comment.